playwright/playwright.config.ts

```diff
@@ -14,10 +14,27 @@ declare const process: {
     MANAGEV2_SECRET: string;
     MANAGEV2_AUDIENCE: string;
     E2E_REPOSITORY: string | undefined;
+    SM_ENV:
+      | "dev-tools"
+      | "marketing-tools"
+      | "platform"
+      | "staging"
+      | "production";
   };
 };
 
```

Comment:
Actually I totally forgot on the previous PR, but we want to have a way to NOT create a repo if we want to run test locally on a yarn play repo for example, right?

Something like: https://github.com/prismicio/customtypes-api/blob/master/__tests__/api/.env.sample
As you see there is a REPOSITORY ENV you can define if you want, and it will use it.

Second thing I found I forgot for the first PR is having actually a file like this, `.sample` so we can copy and paste it and it help us to run things locally.

Reply:
Yeah that makes sense, haven't thought of that.

Just changed to look to `E2E_REPOSITORY` and avoid creating a temporary repo.